devito/types/relational.py

```diff
@@ -74,6 +80,8 @@ class Lt(sympy.Lt):
     >>> Lt(g, 1)
     g(x, y) < 1
     """
+    is_Lte = True
```

Comment:
what are these is_Lte etc flags for?

Reply:
did you mean `isinstance(asm, (Ge, Gt))` ....?

Reply:
did you mean `isinstance(asm, (Ge, Gt))` ....?

Reply:
I think these is_Lte etc are overkill

Reply:
I totally forgot about isinstance... thanks.